sources/packages/web/src/services/http/StudentAssessmentApi.ts

```diff
@@ -56,14 +56,16 @@ export class StudentAssessmentApi extends HttpBaseClient {
    * Get the NOA values for a student application on a particular assessment.
    * @param assessmentId assessment id to get the NOA values.
    * @param studentId student id.
+   * @param applicationId, application id.
    * @returns NOA and application data.
    */
   async getAssessmentNOA(
     assessmentId: number,
     studentId?: number,
+    applicationId?: number,
```

Comment:
Should we maybe change the `studentId` and `applicationId` to be required as below? Same for the others.
```ts
async getAssessmentNOA(
  assessmentId: number,
  options? : {
    studentId: number,
    applicationId: number }
)
```

Reply:
actually, `studentid` and `applicationid` is comming from the common component, where it is actually an optional parameter

Reply:
as discussed keeping as it is